application/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "application"
+version = "0.1.0"
+edition = "2021"
+
```

Comment:
You need to add a license here to shut cargo-deny up. You can also add `LicenseRef-webpki` to the `allow = []` field for cargo deny too as it's got a known funky but valid license that it's moaning about

Reply:
Added the license in `Cargo.toml`. I don't quite get the cargo-deny comment

Reply:
![Screenshot from 2022-04-06 18-14-53](https://user-images.githubusercontent.com/1494620/161995859-8cb8f8a8-d6d0-4be9-8050-c4e78277f6c7.png)

I was talking about fixing that warning to make cargo deny cleaner. It's not important though.

Reply:
I'd forget about it